modules/app/templates/user_data.tpl

```diff
@@ -12,8 +12,7 @@ set :port, 8080
 set :bind, '0.0.0.0'
 
 configure do
-  client = Mongo::Connection.new "${mongo_address}"
```

Comment:
didn't work for me - probably because of newer ruby-mongo lib version installed from 18.04 repos.